index.html

```diff
@@ -7,92 +7,207 @@
-          later let yourself dive into nature. Break the habit.</p>
+          If your job is fully <br> remote and you daydream of holidays, then why <br> sit at home? Work as you normally
+          do and
+          later <br> let yourself dive into nature. Break the habit.</p>
+      </div>
+      <div class="btn_third_sector">
+        <button class="book_now" type="submit"><span class="book">Book Now</span></button>
       </div>
-      <div class="btn"><button>Book now</button></div>
     </div>
     <div class="right_side">
       <img class="mountains_pic" src="mountain_home.png" alt="widok na góry">
     </div>
-  </div>
-  <div class="fourth_page_container">
+  </section>
+
+
+
+
+
+  <section class="section fourth_page_container">
     <div class="left_side_fourth_page">
       <div><img class="girl_pic" src="girl.png" alt="zdjęcie dziewczyny"></div>
-      <div class="newsletter">
-        <label for="newsletter_header"> newsletter comming soon</label>
-        <input type="email" class="email" placeholder="your e-mail address">
-      </div>
+
     </div>
+    
+
+
+
+
+
     <div class="right_side_fourth_page">
-      <h1 class="cabin_lovers">+2,000<br>cabin lovers</h1>
-      <p class="fourth_page_footer">Marina Skovic, Visual Artist</p>
-      <div class="bot_nav_container">
-        <nav class="bot_nav"></nav>
+      <div class="fan_number">
+        <p class="cabin_lovers">+2,000<br>cabin lovers</p>
+      </div>
+      <div class="autor">
+        <footer class="fourth_page_footer">Marina Skovic, Visual Artist</footer>
+        
+        <ul class="unordered_list_for_numbers">
+          <li class="numbers_fourth_page"><a class="anchor_number" href="#">1</li></a>
+          <li class="numbers_fourth_page"><a class="anchor_number" href="#">2</li></a>
+          <li class="numbers_fourth_page"><a class="anchor_number" href="#">3</li></a>
+        </ul>
+      </div></div>
+  </section>
+
+
+
+  <section class="section fifth_page_container">
+    <div class="newsletter">
+      <div class="input_newsletter">
+        <div class="input_header">
+          <label for="newsletter_header" class="newsletter_info"> newsletter comming soon</label>
+        </div>
+        <div class="input_input">
+          <input type="email" class="email" placeholder="your e-mail address">
+        </div>
       </div>
     </div>
-  </div>
 
-</body>
+    <footer class="bot_nav_container">
+      
+        <ul class="ul_container">
+          <li class="nav">Help Center</li>
+          <li class="nav">Facebook</li>
+          <li class="nav">About</li>
+        </ul>
+        <ul class="ul_container">
+          <li class="nav">FAQ</li>
+          <li class="nav">Contact</li>
+          <li class="nav">Instagram</li>
+        </ul>
+        <ul class="ul_container">
+          <li class="nav">Blog</li>
+          <li class="nav">Reviews</li>
+          <li class="nav">Twitter</li>
```

Comment:
delete existing classes form these elements.
Just set:
```css
.ul_container {
  display: flex;
  padding: 0;
  list-style: none;
}

Reply:
and add links like up there! 👆

Reply:
if I do so the whole table is collapsing

Reply:
right, my bad ✋  set flex on the ul's parent

Reply:
it already been there :)